README.md

```diff
@@ -129,13 +131,19 @@ And install the dependencies:
 $ pip install -r requirements-dev.txt
 ```
 
-Run the application with:
+Generally, you can run the application with:
 
 ```sh
 $ python app.py
 ```
 
-And visit `http://localhost:6500/`.
+Then, navigate to `http://localhost:6500/`.
+
+But, if you need to run the application over https, explicitly state that in the command:
```

Comment:
I think it's worth expanding on this a bit, because I want people to understand that there's a pretty specific context where you would do this, and why it works since it's reliant on the Werkzeug implementation (and `ssl_context` doesn't show up in the Flask docs). Something like:

> **Note on HTTP/S in development:** When deployed, the application should be run behind a secure proxy responsible for SSL termination. While developing locally if it becomes necessary to observe app code serving HTTP/S requests, it is possible to start the Flask/Werkzeug development server with an ad-hoc SSL context (see [`werkzeug.serving.run_simple()`](https://werkzeug.palletsprojects.com/en/2.0.x/serving/#werkzeug.serving.run_simple) for more details).
>
> To have the server listen for HTTP/S requests, supply an `https://` URL on start:
>
> ```sh
> $ python app.py https://localhost:6500/
> ```
>
> Also note that this does not fully replicate secure requests as they would appear on a deployed app instance. In particular the `X-Forwarded-*` headers may be different, since you are hitting the application server directly rather than through one or more proxy layers.

Reply:
This should also mention that the adhoc certs also work because of the `pyopenssl` package.

I was originally concerned about this approach because I thought the docker scripts ran `python app.py ...` internally and that nginx's SSL configs would conflict with this local SSL config. It doesn't look like that is the case and there are no issues with the CLI command, right?

Reply:
Right--the actual app runs in Docker via uWSGI knowing where to look for the app object, not via that `run()` command.

Reply:
Thanks for taking a look, both! I have updated the README.